spec/models/petition_spec.rb

```diff
@@ -52,6 +52,30 @@
     it { is_expected.to validate_presence_of(:department).with_message(/must be completed/) }
     it { is_expected.to validate_presence_of(:creator_signature).with_message(/must be completed/) }
 
+    context "sponsor email validations" do
+
+      it 'is valid with 5 sponsor emails' do
+        sponsor_emails = ['test1@test.com', 'test2@test.com', 'test3@test.com', 'test4@test.com', 'test5@test.com']
+        expect(FactoryGirl.build(:petition, sponsor_emails: sponsor_emails)).to be_valid
+      end
+
+      it 'is not valid with less than 5 sponsor emails' do
+        sponsor_emails = ['test1@test.com', 'test2@test.com']
+        expect(FactoryGirl.build(:petition, sponsor_emails: sponsor_emails)).not_to be_valid
+      end
+
+      it 'is not valid with invalid sponsor emails' do
+        sponsor_emails = ['test1test', 'test2@test.com', 'test3@test.com', 'test4@test.com', 'test5@test.com']
+        expect(FactoryGirl.build(:petition, sponsor_emails: sponsor_emails)).not_to be_valid
+      end
+
+      it 'is not valid with duplicate sponsor emails' do
+        sponsor_emails = ['test1@test.com', 'test1@test.com', 'test3@test.com', 'test4@test.com', 'test5@test.com']
```

Comment:
Because the validation that fails here is not about duplicates, but about the number of emails being less than 5.  We should decide what the real rules here are.  Perhaps this test should be ``removes duplicate sponsor emails from the list`` rather than ``is not valid with duplicate sponsor emails``.